src/API.js

```diff
@@ -0,0 +1,41 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xJG2mdwj5m9C9rOtT2IJ/scores/';
+const ul = document.getElementById('names');
+
+const displayScores = (list) => {
+  ul.textContent = '';
+  list.forEach((score) => {
+    const li = document.createElement('li');
+    const userName = document.createElement('span');
+    const userScore = document.createElement('span');
+    userName.innerText = `${score.user} :`;
+    userScore.innerText = score.score;
+    userName.className = 'span';
+    userScore.className = 'span';
+    li.append(userName, userScore);
+    ul.appendChild(li);
+  });
+};
+
+const getScores = async () => {
+  const response = await fetch(url);
+  const scores = await response.json();
+  displayScores(scores.result);
+};
+
+const addScore = async () => {
+  (await fetch(url, {
+    method: 'POST', // or 'PUT'
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: document.querySelector('#name').value,
+      score: document.querySelector('#score').value,
+    }),
+  })).json();
```

Comment:
[OPTIONAL] I think you could validate either through HTML or JS that the user can't send empty values. This leads to a better user experience and potentially avoiding bugs in the future.